### ISSUE TITLE
`Command 'rbenv' not found` after a fresh install of...everything

### ISSUE THREAD

**Issue by pil0u** [lewagon/setup#312]
Hello team, I am setting up a Virtual Machine to teach classes in a clean environment as close as the one from the students.
The steps I followed so far:
1. Use the latest VirtualBox version (6.1.22)
2. Create a VM with the latest Ubuntu LTS version (20.04.2.0)
3. Carefully follow [the setup page for Linux](https://github.com/lewagon/setup/blob/master/ubuntu.md)

Everything went smoothly until the [rbenv section](https://github.com/lewagon/setup/blob/master/ubuntu.md#rbenv).
`rvm` was not previously installed, neither was `rbenv`.
I run these command lines smoothly:
``` bash
sudo apt install -y build-essential tklib zlib1g-dev libssl-dev libffi-dev libxml2 libxml2-dev libxslt1-dev libreadline-dev
git clone https://github.com/rbenv/rbenv.git ~/.rbenv
git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
```
But as soon as I try to run the `rbenv ` command, I get:
```
Command 'rbenv' not found, but can be installed with:

sudo apt install rbenv
```

Running the suggested command solves the problem.

**Comment by pil0u** [comment]
After some digging, I realized that `zsh` was not the default terminal, while I correctly said "Yes" when asked for the switch.
As a consequence, the dotfiles were not properly loaded, `rbenv` was not in the `PATH`.

What is weird is that I had no apparent error following the tutorial.

However, "everything went smoothly" might not be totally true: I have my own `dotfiles` personal repo, so when I ran the command that tried to fork Le Wagon's repo, the forked repo was automatically renamed to `dotfiles_lewagon`. For some reasons, I had to handle merge conflicts with that repo, which I found very weird as it was just being created.

I removed the created symlinks with that repo, I deleted entirely that repo from GitHub and locally, I uninstalled `rbenv` via `apt` and I restarted from this command:
``` bash
gh repo fork lewagon/dotfiles --clone
```
This time, the repo was renamed `dotfiles-1` (which btw makes more sense :thinking:), and everything went well. `zsh` is the default terminal and it properly loads its dotfile, making `rbenv` a proper terminal command.

Sorry for the inconvenience.
